sites/cheerpx/src/content/docs/20-faq.md

```diff
@@ -54,3 +54,17 @@ Cross-Origin-Opener-Policy: same-origin
 ```
 
 By adding these headers to your server configuration you will enable cross-origin isolation and CheerpX will be able to start.
+
+## Why Can't I Use the `file://` Protocol?
+
+CheerpX requires certain browser capabilities that are not available when accessing files directly using the `file://` protocol. Here’s an explanation of the issue and the recommended solution:
+
+### Security Restrictions and Cross-Origin Headers
+
+Modern browsers enforce strict security policies to protect users from potentially harmful actions. Accessing files through the `file://` poses a security risk, as it could allow untrusted HTML files to interact with local resources. For instance, if you open a random HTML file received via email, it could potentially access sensitive files on your disk, leading to data breaches.
+
+Additionally, CheerpX relies on cross-origin isolation, which requires specific HTTP headers that can only be set when serving content over HTTP or HTTPS. Without these headers, the necessary security mechanisms cannot be enforced. This is particularly important for CheerpX, as it needs to download multiple resources from its origin, including the WASM file.
```

Comment:
The requirement of downloading multiple resource should be moved to the previous paragraph.